packages/core/contracts/oracle/interfaces/SkinnyOptimisticOracleInterface.sol

```diff
@@ -0,0 +1,240 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../interfaces/OptimisticOracleInterface.sol";
+
+/**
+ * @title Interface for the gas-cost-reduced version of the OptimisticOracle.
+ * @dev Interface used by financial contracts to interact with the Oracle. Voters will use a different interface.
+ */
+abstract contract SkinnyOptimisticOracleInterface {
+    // Struct representing a price request. Note that this differs from the OptimisticOracleInterface's Request struct
+    // in that refundOnDispute is removed.
+    struct Request {
+        address proposer; // Address of the proposer.
+        address disputer; // Address of the disputer.
+        IERC20 currency; // ERC20 token used to pay rewards and fees.
```

Comment:
is there any gas difference with storing this as an address vs an interface like this?

Reply:
Not sure, but i kept this to maintain as much consistency with the OO implementation as possible

Reply:
ok cool.